_sources/SimplePythonData/Values.rst

```diff
@@ -0,0 +1,51 @@
+..  Copyright (C)  Brad Miller, David Ranum, Jeffrey Elkner, Peter Wentworth, Allen B. Downey, Chris
+    Meyers, and Dario Mitchell.  Permission is granted to copy, distribute
+    and/or modify this document under the terms of the GNU Free Documentation
+    License, Version 1.3 or any later version published by the Free Software
+    Foundation; with Invariant Sections being Forward, Prefaces, and
+    Contributor List, no Front-Cover Texts, and no Back-Cover Texts.  A copy of
+    the license is included in the section entitled "GNU Free Documentation
+    License".
+
+.. qnum::
+   :prefix: data-2-
+   :start: 1
+
+Values and Data Types
+---------------------
+
+A **value** is one of the fundamental things --- like a word or a number ---
+that a program manipulates. The values we have seen so far are ``5`` (the
+result when we added ``2 + 3``), and ``"Hello, World!"``.  
+
+We can specify values directly in the programs we write. For example we can specify a number as a literal just by writing it directly, (e.g., ``5`` or ``4.32``). In a program, we specify a word, or more generally a **string** of characters, by enclosing the characters inside quotation marks (e.g., ``"Hello, World!"``).
+
+During execution of a program, the python interpreter creates an internal representation of values that are specified in a program. It can then manipulate them. For example, in the next section, we'll look at how a program can specify that values should be combined using operators like ``+`` and ``*``. We call the internal representations **objects** or just **values**. 
+
+.. note:
+   When we are being careful, we will refer to a number or string that is specified directly in a program as a **literal**, and use the word **value** to refer to the Python interpreter's internal representation of the number or string during the execution of the program. Sometimes, however, we will get a little sloppy and refer to literals as values. It may help you to keep in mind the distinction between a value as written in a program (a literal) and the internal representation of a value.
+
+Normally, people don't get to see directly what is happening when a program executes. Codelens is really useful for learning because it does make a lot of things visible during a program execution.
```

Comment:
I think we need to actually define type in this section.  At the end we say classes and types are interchangeable but we never define type.

These objects are classified into different classes, or data types: 4 is an integer, and "Hello, World!" is a string, so-called because it contains a string or sequence of letters. You (and the interpreter) can identify strings because they are enclosed in quotation marks.